app/models/appointment.rb

```diff
@@ -0,0 +1,7 @@
+class Appointment < ApplicationRecord
+  has_many :appointment_holds
+  has_many :holds, through: :appointment_holds
+  belongs_to :member
+
+  validates :starts_at, :ends_at, presence: true
```

Comment:
I think we should add one more validation on the `starts_at` and `ends_at` date ranges, to make sure the `starts_at` is earlier, like this:

```
validate :ends_at_later_than_starts_at

private
  def ends_at_later_than_starts_at
    if ends_at < starts_at
      errors.add(:ends_at, "must be after the starts_at date")
    end
 end
```